api/src/apollo/EventRegistration/mutations.ts

```diff
@@ -0,0 +1,54 @@
+import { idArg, inputObjectType, mutationField, nonNull } from 'nexus';
+
+export const EventRegistrationCreateInputType = inputObjectType({
+  name: 'EventRegistrationCreateInputType',
+  definition(t) {
+    t.nonNull.id('eventID');
+    t.nonNull.id('userID');
+  },
+});
+
+export const createEventRegistration = mutationField(
+  'createEventRegistration',
+  {
+    type: 'EventRegistration',
+    args: {
+      eventRegistration: nonNull('EventRegistrationCreateInputType'),
+    },
+    resolve(_parent, args, { prisma }) {
+      return prisma.eventRegistration.create({
+        data: args.eventRegistration,
+      });
+    },
+  },
+);
+
+export const EventRegistrationUpdateInput = inputObjectType({
+  name: 'EventRegistrationUpdateInput',
+  definition(t) {
+    t.id('eventID');
+    t.id('userID');
+  },
+});
+
+export const updateEventRegistration = mutationField(
```

Comment:
There won't be any case of updateEventRegistration, we can emit this

Reply:
not even from the dashboard and all? in case of fests